### PR TITLE
cpulimit: add page

### PR DESCRIPTION
**Pull request by slicer69**
Add TLDR page for cpulimit command.

- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/cpulimit.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# cpulimit
+
+> A tool to throttle the CPU usage of other processes.
+> Homepage: <http://limitcpu.sourceforge.net/>.
+
+- Limit an existing process with PID 1234 to only use 25% of the CPU:
+
+`cpulimit -p 1234 -l 25%`
```
Comment (pxgamer) [inline pages/linux/cpulimit.md]:
```suggestion
`cpulimit -p {{1234}} -l {{25%}}`
```

File: pages/linux/cpulimit.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Limit an existing program by its executable name instead of PID number:
+
+`cpulimit -e {{program}} -l 25`
```
Comment (pxgamer) [inline pages/linux/cpulimit.md]:
```suggestion
`cpulimit -e {{program}} -l {{25}}`
```

File: pages/linux/cpulimit.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Launch a given program and limit it to only use 50% of the CPU:
+
+`cpulimit -l 50 {{program}}`
```
Comment (pxgamer) [inline pages/linux/cpulimit.md]:
```suggestion
`cpulimit -l {{50}} {{program}}`
```

File: pages/linux/cpulimit.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Launch a program, limit its CPU usage to 50% and run cpulimit in the background:
+
+`cpulimit -l 50 -b {{program}}`
```
Comment (pxgamer) [inline pages/linux/cpulimit.md]:
```suggestion
`cpulimit -l {{50}} -b {{program}}`
```

File: pages/linux/cpulimit.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Launch a program and kill its process if the program's CPU usage goes over 50%:
+
+`cpulimit -l 50 -k {{program}}`
```
Comment (pxgamer) [inline pages/linux/cpulimit.md]:
```suggestion
`cpulimit -l {{50}} -k {{program}}`
```

File: pages/linux/cpulimit.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Launch a program and throttle both it and its child processes so that none go about 25% CPU:
+
+`cpulimit -l 25 -m {{program}}`
```
Comment (pxgamer) [inline pages/linux/cpulimit.md]:
```suggestion
`cpulimit -l {{25}} -m {{program}}`
```

**Comment by sbrl** [comment]
I'll tidy this up & create a new PR.

**Comment by pxgamer** [comment]
To be fair, it may have just been easier to commit the suggested changes and merge this PR. 🤔

They were pretty trivial changes.

**Comment by sbrl** [comment]
I was going to, @pxgamer, but it doesn't seems to want to let me do that :thinking: 

image

**Comment by pxgamer** [comment]
Ah, I see. Didn't realise that they'd locked the PR. 👍

**Pull request closed without merging**